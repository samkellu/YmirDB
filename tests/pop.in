pop a
set a
pop a
set b 12
set a 123 44 b
get a
pop a
get a
pop a
get a
pop a
get a
pop a
get a
get b
bye
