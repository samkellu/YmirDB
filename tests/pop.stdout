==834== Memcheck, a memory error detector
==834== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==834== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==834== Command: ./ymirdb
==834== 
==834== error calling PR_SET_PTRACER, vgdb might block
> no such key

> ok

> nil

> ok

> ok

> [123 44 b]

> 123

> [44 b]

> 44

> [b]

> b

> []

> nil

> []

> [12]

> 
bye
==834== 
==834== HEAP SUMMARY:
==834==     in use at exit: 0 bytes in 0 blocks
==834==   total heap usage: 61 allocs, 61 frees, 2,232 bytes allocated
==834== 
==834== All heap blocks were freed -- no leaks are possible
==834== 
==834== For lists of detected and suppressed errors, rerun with: -s
==834== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
